### ISSUE TITLE
Dependabot `path-to-regexp` report due to `serve` usage

### ISSUE THREAD

**Issue by MikeMcC399** [cypress-io/cypress-example-kitchensink#889]
## Issue

Dependabot reports a high severity vulnerability [path-to-regexp outputs backtracking regular expressions](https://github.com/advisories/GHSA-9wv6-86v2-598j) in this repo concerning [CVE-2024-45296](https://nvd.nist.gov/vuln/detail/CVE-2024-45296).

This is also reported by `npm audit`:

```text
path-to-regexp  0.2.0 - 7.2.0
Severity: high
path-to-regexp outputs backtracking regular expressions - https://github.com/advisories/GHSA-9wv6-86v2-598j
fix available via `npm audit fix --force`
Will install serve@6.5.8, which is a breaking change
node_modules/serve-handler/node_modules/path-to-regexp
  serve-handler  *
  Depends on vulnerable versions of path-to-regexp
  node_modules/serve-handler
    serve  >=7.0.0
    Depends on vulnerable versions of serve-handler
    node_modules/serve
```

The vulnerability is pulled in by

https://github.com/cypress-io/cypress-example-kitchensink/blob/31bf6677fb3ddb7e9cc142ddf5e8dcf6821fa76e/package.json#L64

```text
$ npm ls path-to-regexp
cypress-example-kitchensink@0.0.0-development
└─┬ serve@14.2.3
  └─┬ serve-handler@6.1.5
    └── path-to-regexp@2.2.1
```

- The issue has been reported to https://github.com/vercel/serve/issues/811 ~~, however at this time there is no resolution available.~~

**Comment by MikeMcC399** [comment]
- A fix is now available through [serve@14.2.4](https://github.com/vercel/serve/releases/tag/14.2.4).

- The update can be triggered through dependabot by somebody with write access to the repo's issue https://github.com/cypress-io/cypress-example-kitchensink/issues/176

![image](https://github.com/user-attachments/assets/8854937e-860b-4ed7-a251-b25a5fa9ff2b)

**Comment by MikeMcC399** [comment]
- PR https://github.com/cypress-io/cypress-example-kitchensink/pull/897 should resolve this issue.
